docs/examples/cookbook/drag-and-drop.md

```diff
@@ -182,7 +182,7 @@ local function TodoEntry(
 	}
 
 	local itemPosition = scope:Value(nil)
-	local itemIsDragging = scope:Computed(function(use)
+	local itemIsDragging = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
	local itemIsDragging = scope:Computed(function(use)
```

Reply:
reverted